### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by dennisdolliver**
## Summary

Adds in a Layout section to the foundation with two functional pieces:

- Aspect-ratio
- Object-fit

This is a port of functionality from the marketing kyndryl storybook

### CODE REVIEW

**Review by brian-patrick-3 — CHANGES_REQUESTED** [review]
Looking pretty solid! Just a few minor comments.

File: src/scss/utility/layout.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@use '../mixins/layout.scss';
+
+.kd-layout--aspect-ratio {
```
[the remark this answers is not in the document]
Comment (dennisdolliver) [reply src/scss/utility/layout.scss]: Addressed in the latest commits

File: src/stories/Layout.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { Meta, Story, Canvas, Controls, Stories } from '@storybook/blocks';
+import * as LayoutStories from './Layout.stories.js';
+
+<Meta of={LayoutStories} />
+
+# Layout
+
+Work in progress
```
[the remark this answers is not in the document]
Comment (dennisdolliver) [reply src/stories/Layout.mdx]: Addressed in the latest commits

File: src/stories/Layout.stories.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+export default {
+  title: 'Foundation/Layout',
+
+  decorators: [
+    (story) =>
+      html`
+        <style>
+          .layout-examples .aspect-ratio-box {
+            width: 500px;
```
[the remark this answers is not in the document]
Comment (dennisdolliver) [reply src/stories/Layout.stories.js:11]: Addressed in the latest commits

**Review by brian-patrick-3 — APPROVED** [review]

**Pull request merged**